cpu/cortexm_common/include/cpu.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2014 Freie Universität Berlin
+ * Copyright (C) 2015 Freie Universität Berlin
```

Comment:
Maybe I was wrong about the MUST, but it seems to be a good idea not to change, but to expand, see http://stackoverflow.com/a/2391555 .